ReactWindows/ReactNative.Shared/UIManager/TransformHelper.cs

```diff
@@ -119,7 +120,7 @@ private static double ConvertToRadians(JObject transformMap, string key)
                     stringValue = stringValue.Substring(0, stringValue.Length - 3);
                 }
 
-                value = double.Parse(stringValue);
+                value = double.Parse(stringValue, CultureInfo.InvariantCulture);
```

Comment:
While this was not the cause of the FormatException it is subjected to the same failure if the angle used contains a fractional part.  So adding the invariant culture qualifier here as well.